nexus/client.go

```diff
@@ -259,6 +260,9 @@ type ExecuteOperationOptions struct {
 	// Even though Client.ExecuteOperation waits for operation completion, some applications may want to set this
 	// callback as a fallback mechanism.
 	CallbackURL string
+	// Optional header fields set by a client that are required to be attached to the callback request when an
+	// asynchronous operation completes.
+	CallbackHeader Header
```

Comment:
If there can only be one value per key here, consider just using a map.

Reply:
Header has a case insensitive getter, it's convenient.